locate/word_test.go

```diff
@@ -267,13 +277,21 @@ func TestNextWordEnd(t *testing.T) {
 			count:       3,
 			expectedPos: 16,
 		},
+		{
+			name:        "count three, with punctuation",
+			inputString: "lorem, ipsum, dolor, sit, amet",
+			withPunct:   true,
+			pos:         2,
+			count:       3,
+			expectedPos: 19,
+		},
```

Comment:
would be great to add a few more test cases here (similar to the other cases involving punctuation above)

Reply:
this should be done now